lib/create-testing-library-rule/detect-testing-library-utils.ts

```diff
@@ -299,15 +301,20 @@ export function detectTestingLibraryUtils<
 
 		// Helpers for Testing Library detection.
 		const getTestingLibraryImportNode: GetTestingLibraryImportNodeFn = () => {
-			return importedTestingLibraryNode;
+			return importedTestingLibraryNodes[0];
```

Comment:
So clever! I was struggling to implement this change since I didn't want to refactor all this logic. Leaving it pointing to the first module found is a great workaround!

Reply:
Thank you so much! I thought that this is the easiest workaround and hoped that it's enough for the reviewer(s) too 🤗  I am happy to look at those other rules that still use the old util function, because I got stuck with them too 😅 But maybe together we will untangle that mess 💪

Reply:
We can tackle that in a separate PR for sure, thanks!